src/Api/Api.php

```diff
@@ -170,7 +176,8 @@ public function __construct(
         array $errors = [],
         array $providers = [],
         string $connection = null,
-        bool $transactions = true
+        bool $transactions = true,
+        string $modelNamespace = null
```

Comment:
14 params for constructor... 
Maybe we have the better way?

Reply:
well it was 13! :) . However, if this is not acceptable, I think I can make the model namespace as an optional argument in the command itself, so something in the lines of "make:json-api:resource {model-namespace? : The Namespace of the model this resource is connected to.}" 
and eliminate the default in the configuration. (the 14th parameter)

Reply:
No it's fine for the moment... as in it's not fine, I need to refactor - but not really the purpose of this PR to refactor it. So it's ok to add a 14th argument now and I'll sort out the code-smell later on!